src/pages/AppConfigure.vue

```diff
@@ -13,13 +13,20 @@ import {
   setAppName,
   setClientIds,
   setDefaultLimit,
+  enableUiMode,
 } from '@/services/smart-contract.service'
-import type { ConfigureTab, ConfigureTabType } from '@/utils/constants'
+import {
+  WalletModeMapping,
+  type ConfigureTab,
+  type ConfigureTabType,
+} from '@/utils/constants'
 
 const currentTab: Ref<ConfigureTabType> = ref('general')
 const router = useRouter()
 const store = useStore()
 
+let currentConfig: AppConfig = store.getters.appConfigRequestBody
```

Comment:
```suggestion
let currentConfig: AppConfig = computed(() => store.getters.appConfigRequestBody)
```

Reply:
We don't need this to be reactive, just need the current snapshot of the state